svf/include/Util/SVFStat.h

```diff
@@ -30,6 +30,8 @@
 #ifndef SVF_SVFSTAT_H
 #define SVF_SVFSTAT_H
 
+#include <string>
```

Comment:
Do we need to include the string header here? It would be good to put in the cpp file.